01_introduction.Rmd

```diff
@@ -2,23 +2,76 @@
 
 **Learning objectives:**
 
-- THESE ARE NICE TO HAVE BUT NOT ABSOLUTELY NECESSARY
+- What this book is about! 
```

Comment:
```suggestion
- Describe what this book is about! 
```

In general, aim for learning objectives to complete the sentence "After you read this chapter, you should be able to..."